Sources/FusionBluetooth_Common/BluetoothManager.swift

```diff
@@ -0,0 +1,152 @@
+import Foundation
+
+/**
+ *  @struct Peripheral
+ *
+ *  @discussion The device data for peripheral
+ *
+ */
+public struct Peripheral: Equatable {
+
+    /**
+     *  @property name
+     *
+     *  @discussion The device name.
+     *
+     */
+    public let name: String?
+    
+    /**
+     *  @property uuid
+     *
+     *  @discussion The identifier in iOS and MAC address in Android.
+     *
+     */
+    public let uuid: String
+    
+     /**
+     *  @property isConnected
+     *
+     *  @discussion The state whether the peripheral is connected or not
+     *
+     */
+    public var isConnected: Bool
+    
+    public init(name: String?, uuid: String, isConnected: Bool) {
```

Comment:
I would make isConnected a function so that one can check the status any time, also, I would move all functions that work with a device to this struct and maybe call it Device. See the next comment.